src/celeritas/phys/PhysicsTrackView.hh

```diff
@@ -153,6 +153,11 @@ class PhysicsTrackView
     template<class T>
     inline CELER_FUNCTION T make_calculator(ValueGridId) const;
 
+    // Construct a grid calculator from a physics table
+    template<class T>
+    inline CELER_FUNCTION T make_calculator(ValueGridId,
+                                            const size_type order) const;
```

Comment:
Generally we don't `const` local function arguments, particularly because this can affect Windows ABI
```suggestion
                                            size_type order) const;
```